common/djangoapps/third_party_auth/strategy.py

```diff
@@ -57,3 +60,11 @@ def setting(self, name, default=None, backend=None):
         # At this point, we know 'name' is not set in a [OAuth2|LTI|SAML]ProviderConfig row.
         # It's probably a global Django setting like 'FIELDS_STORED_IN_SESSION':
         return super(ConfigurationModelStrategy, self).setting(name, default, backend)
+
+    def request_data(self, merge=True):
+        """
+        process the request data. Overriding it to log the data values.
+        """
+        data = super(ConfigurationModelStrategy, self).request_data(merge=merge)
+        log.info('ConfigurationModelStrategy: processed request data "%s" ', data)
```

Comment:
I don't think we want to log this here.

Reply:
I think, adding log here would be helpful to analyze the data b/w different calls before jumping to auth_complete procedure.